projects.json

```diff
@@ -466,6 +466,7 @@
       "Fuel",
       "Tooling"
     ],
+    "showAccountTag": true,
```

Comment:
will this have any effect as this project doens't have `address` ?

Reply:
This has no effect; I have removed it, it's a remnant from a previous test I conducted locally. Thank you

Reply:
This has no effect; I have removed it, it's a remnant from a previous test I conducted locally. Thank you